packages/core/src/amazonqFeatureDev/session/session.ts

```diff
@@ -166,8 +166,13 @@ export class Session {
         return resp.interaction
     }
 
-    public async updateFilesPaths(tabID: string, filePaths: NewFileInfo[], deletedFiles: DeletedFileInfo[]) {
-        this.messenger.updateFileComponent(tabID, filePaths, deletedFiles)
+    public async updateFilesPaths(
+        tabID: string,
+        filePaths: NewFileInfo[],
+        deletedFiles: DeletedFileInfo[],
+        messageId: string
```

Comment:
There's now 4 complex args for this function, and multiple other functions in this file use the same set of args.
To help with readability, consider passing a typed args object to each function with the same set of arguments instead, e.g.

```
type UpdateFileOpts {
  tabId: string;
  filePaths: NewFileInfo[];
  deletedFiles: DeletedFileInfo[];
  messageId: string;
}

public async updateFilesPaths(opts: UpdateFileOpts) {
   ...
}

updateFilesPaths({
  tabId: this.tabId,
  filePaths,
  deletedFiles,
  messageId: this.messageId
})
```

Reply:
if its ok I will add it as a follow up.